src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/_help.py

```diff
@@ -70,124 +70,234 @@
 helps['eventgrid event-subscription'] = """
     type: group
     short-summary: Manage event subscriptions for an Event Grid topic or for an Azure resource.
+    long-summary: Manage event subscriptions for an Event Grid topic, Azure subscription, resource group or for any other Azure resource that supports event notifications.
     """
 helps['eventgrid event-subscription create'] = """
     type: command
     short-summary: Create a new event subscription for an Event Grid topic or for an Azure resource.
+    parameters:
+        - name: --source-resource-id
```

Comment:
Thanks for the suggestion @tjprescott . While we understand the goal of this suggestion we feel it will harm our users experience who have been used to this name for quite some time now. We initially have resource-id and then per our discussion with you couple of months ago we moved to the new name source-resource-id and it has been used by customer for some time now. We would like to keep this name for user convenience and reduce any confusion.